docs/index.md

```diff
@@ -14,8 +14,9 @@ We assume that users installing and using Clinica are comfortable using the comm
 
 <!-- ### Installing Clinica from source -->
 
-- [Installation](./Installation)
-- [Third-party software](./Third-party)  
+- [Installation](Software/Installation.md)
```

Comment:
I know it is not related to what you changed, but it could be the opportunity to change it:

Maybe we could remove the link on the title of the first section: What is Clinica and have a one sentence description with a link to learn more. Basically something like:

```markdown
# Clinica Documentation

## What is Clinica?

Clinica is a software platform for clinical neuroscience research studies using multimodal data and most often longitudinal follow-up. You can learn more on [this page](WhatIsClinica).

## Installation

Clinica can be installed....
```

WDYT ?

Reply:
Sure, I can do that !